schemas/groups/notifications.json

```diff
@@ -4,14 +4,19 @@
   "id": "https://signalk.github.io/specification/schemas/groups/notifications.json#",
   "title": "notifications",
   "description": "Notifications, their state, and actions. The notification limits are set in any Signal K key.meta.zones array.",
-
+  "dependencies": {
+  	"message": ["state"]
```

Comment:
Could you elaborate on this - none of the properties are now required, but if there is `message` then `state` must also be there.